Public/Sdk/SelfHost/BuildXL/Tools/QTest/Tool.QTestRunner.dsc

```diff
@@ -188,7 +188,7 @@ export function runQTest(args: QTestArguments): Result {
     let result = Transformer.execute({
         tool: args.qTestTool ? args.qTestTool : qTestTool,
         tags: tags,
-        description: args.description,
+        description: `${args.description} || Running QTest for ${args.testAssembly}`,

```

Comment:
Using format string is not needed....
The format quote should be after ||

Reply:
I actually want both of them to be in the description. But since args.description can be none I will put it in the back so it will look nicely.